src/client/game.cpp

```diff
@@ -2885,20 +2885,9 @@ void Game::handleClientEvent_PlayerForceMove(ClientEvent *event, CameraOrientati
 	cam->camera_pitch = event->player_force_move.pitch;
 }
 
-void Game::handleClientEvent_Deathscreen(ClientEvent *event, CameraOrientation *cam)
+void Game::handleClientEvent_DeathscreenLegacy(ClientEvent *event, CameraOrientation *cam)
 {
-	// If client scripting is enabled, deathscreen is handled by CSM code in
-	// builtin/client/init.lua
-	if (client->modsLoaded())
-		client->getScript()->on_death();
-	else
-		showDeathFormspec();
-
-	/* Handle visualization */
-	LocalPlayer *player = client->getEnv().getLocalPlayer();
-	runData.damage_flash = 0;
-	player->hurt_tilt_timer = 0;
-	player->hurt_tilt_strength = 0;
```

Comment:
Since the new death screen is just a regular formspec, doing special stuff like this will no longer be possible. This could however be just fine, since it looks like the damage flash isn't applied on death anyway:

https://github.com/minetest/minetest/blob/733a019bf5aefddb824e5643b791ca25fc620b03/src/client/game.cpp#L2862-L2876